src/utils/util.js

```diff
@@ -16,9 +16,17 @@ const getShopifyUserInfo = require('./getShopifyUserInfo')
 const API_VERSION = '1.0'
 
 const s3Config = {}
-if(process.env.AWS_KEY && process.env.AWS_SECRET) {
-  s3Config.accessKeyId = process.env.AWS_KEY
-  s3Config.secretAccessKey = process.env.AWS_SECRET
+if(process.env.AWS_ACCESS_KEY_ID && process.env.AWS_SECRET_ACCESS_KEY) {
+  s3Config.accessKeyId = process.env.AWS_ACCESS_KEY_ID
+  s3Config.secretAccessKey = process.env.AWS_SECRET_ACCESS_KEY
+} else if (process.env.S3_AWS_ACCESS_KEY_ID && process.env.S3_AWS_SECRET_ACCESS_KEY) {
+  s3Config.accessKeyId = process.env.S3_AWS_ACCESS_KEY_ID
+  s3Config.secretAccessKey = process.env.S3_AWS_SECRET_ACCESS_KEY
+}
```

Comment:
Please either revert this or rename to `S3_ACCESS_KEY_ID` and `S3_SECRET_ACCESS_KEY` and apply the same to all references